datalad/support/sshconnector.py

```diff
@@ -118,6 +119,28 @@ def close(self):
             else:
                 raise
 
+    def copy(self, source, destination):
+        """Copies source file/folder to destination on the remote.
+
+        Parameters
+        ----------
+        source: str
+          file/folder path to copy from on local
+        destination: str
+          file/folder path to copy to on remote
+
+        Returns
+        -------
+        str
+          stdout, stderr of the copy operation.
+        """
+
+        self.scp_prefix = ["scp", "-o", "ControlPath=" + self.ctrl_path]
```

Comment:
I think we should refactor `__init__`, to store 
```python
self.cmd_options = ["-o", "ControlPath=%s" % self.ctrl_path]
```
instead of `self.cmd_prefix` and then specify explicitly command (ssh) in `__call__` and then reuse cmd_options in both call and copy